pnv/.htaccess

```diff
@@ -0,0 +1,26 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+
+RewriteEngine on
+
+#Rewrite rules for pnv vocabulary 
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^pnv$ http://www.lodewijkpetram.nl/vocab/pnv/pnv.ttl  [R=303,NE,L]
```

Comment:
These rules are relative to the current directory so would result in `https://w3id.org/pnv/pnv`.  You probably mean to use `^$`.

Reply:
Yes, you're right. I updated the .htaccess.

Sorry for the extra work,
Lodewijk